src/components/VideoCardSmall/VideoCardSmall.component.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { useHistory } from 'react-router';
+import {
+  CardSmall,
+  CardContainerSmall,
+  CardImageSmall,
+  CardContentSmall,
+  TitleSmall,
+} from './VideoCardSmall.styles';
+
+export default function VideoCardSmall({ video: { id, channelTitle, image, title } }) {
+  const { theme } = useContext(ThemeContext);
+  const history = useHistory();
+  const openVideoPage = () => {
+    history.push(`/watch/${id}`);
+  };
+
+  return (
+    <CardSmall theme={theme}>
+      <CardContainerSmall onClick={openVideoPage}>
```

Comment:
Same recommendation here, you could use `Link` component from `react-router-dom` instead of onClick event